packaging/docker/bootstrap_scripts/balloon.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+cwd=$(pwd)
+echo "# INSTALL & CONFIGURE BALLOON"
+echo "## INSTALL BUILD DEPENDENCIES"
+
+export BALLOON_VERSION=$(cat /srv/www/balloon/VERSION)
+if [ ${BALLOON_VERSION:0:1} -lt 2 ]
+then
+    echo "ERROR: Dockerfile requires a balloon version >= 2.0.0"
+    exit 1
+fi
+
+export DEBIAN_FRONTEND=noninteractive
+apt-get update
+apt-get --no-install-recommends install -y git npm make
+# create node alias for nodejs binary
+update-alternatives --install /usr/bin/node node /usr/bin/nodejs 10
+npm -g install bower apidoc
+
+echo "## INSTALL COMPOSER"
+php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
+php -r "if (hash_file('SHA384', 'composer-setup.php') === '544e09ee996cdf60ece3804abc52599c22b1f40f4323403c44d44fdfdd586475ca9813a858088ffbc1f233e9b180f061') { echo 'Installer verified'; } else { echo 'Installer corrupt'; unlink('composer-setup.php'); } echo PHP_EOL;"
+php composer-setup.php --filename=composer --install-dir=bin
+php -r "unlink('composer-setup.php');"
+
+echo "# INSTALL BALLOON"
+balloonDir="/srv/www/balloon"
+cd $balloonDir
+
+echo "## BUILD BALLOON"
+# build balloon
+make build
+
+cd $cwd
+
+# create self-signed ssl certificate
```

Comment:
if $DEV then it will probably make sense to leave out ssl or better to add a env variable SELF_SIGNED_CERT which default is no.

Reply:
or maybe just SSL=yes|no

Reply:
postponed